src/neo4j_genai/generation/text_splitters.py

```diff
@@ -0,0 +1,42 @@
+#  Neo4j Sweden AB [https://neo4j.com]
```

Comment:
For both the text splitters and the kg writer, do we want these under src/neo4j_genai/generation/? I had thought this folder is meant for the generation features for RAG

I think it would make more sense to put these under a different folder src/neo4j_genai/kg_construction/